src/test/javascript/spec/component/programming-exercise/programming-exercise-update.component.spec.ts

```
@@ -940,6 +940,49 @@ describe('ProgrammingExerciseUpdateComponent', () => {
                 });
             }
         });
+
+        it('should find invalid timeout', () => {
+            comp.programmingExercise.buildConfig!.timeoutSeconds = -1;
```

Biome

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)